packages/dds/tree/src/shared-tree/sharedTree.ts

```diff
@@ -382,21 +373,7 @@ export class SharedTreeKernel
 	}
 
 	public exportVerbose(): VerboseTree | undefined {
-		const cursor = this.checkout.forest.allocateCursor("contentSnapshot");
-		try {
-			moveToDetachedField(this.checkout.forest, cursor);
-			const length = cursor.getFieldLength();
-			if (length === 0) {
-				return undefined;
-			} else if (length === 1) {
-				cursor.enterNode(0);
-				return verboseFromCursor(cursor, this.storedSchema.nodeSchema);
-			} else {
-				fail(0xac8 /* Invalid document root length */);
-			}
-		} finally {
-			cursor.free();
-		}
+		return this.checkout.exportVerbose();
```

Comment:
The implementation of exportVerbose was moved to the checkout so that more test code can access it.